docs/2.5.1.md

```diff
@@ -0,0 +1,36 @@
+# ProxySQL 2.5.1 and ProxySQL admin tools (2023-04-)
+
+| Release date | April , 2023 |
+|---|---|
+| Install instructions | [ProxySQL and proxysql-admin](https://docs.percona.com/proxysql/install-v2.html) |
+| Download this version | [ProxySQL 2](https://www.percona.com/download-proxysql) using the [instructions](https://docs.percona.com/proxysql/where-to-download-proxysql.html)|
+
+[ProxySQL](https://proxysql.com/) is a high-performance proxy for MySQL and MySQL-compatible database servers such as Percona Server for MySQL and MariaDB. It acts as an intermediary for client requests seeking resources from the database. ProxySQL was created for the database administrator as a solution for complex replication topology issues. 
+
+You can [download the original ProxySQL from GitHub](https://github.com/sysown/proxysql/releases). The [ProxySQL documentation](https://proxysql.com/documentation/) provides information on installing and running ProxySQL. 
+
+The [ProxySQL Admin (proxysql-admin)](proxysql-admin-tool-v2-config.md) tool simplifies the configuration of Percona XtraDB Cluster nodes with ProxySQL. [ProxySQL Admin 1.x.x](proxysql-v1.md) requires custom `Bash` scripts to track the status of a Percona XtraDB Cluster. [ProxySQL Admin 2.x.x](proxysql-admin-tool-functions.md) supports Percona XtraDB Cluster without custom scripts.
+
+The [ProxySQL 2.x.x and pxc_scheduler_handler](psh-overview.md) tool can automatically perform a failover due to node failures, service degradation, or maintenance. The pxc_scheduler_handler tool has different features and capabilities than ProxySQL admin. Do not use the options from one tool in the other tool. Mixing the options may cause unintended results.
+
+## Release highlights
+
+This release includes all enhancements and bug fixes available in ProxySQL 2.5.1. The improvements and bug fixes introduced by ProxySQL and included in this release are:
```

Comment:
IMO it is better we even include the changes from 2.5.0 as we are releasing 2.5.1 which also includes changes from 2.5.0. Link is [here](https://github.com/sysown/proxysql/releases/tag/v2.5.0).

Also it would be better to start the highlights with something like below
```
Compared to previous release 2.4.8, ProxySQL 2.5.1 introduces the following changes:
```